src/Castle.Core/DynamicProxy/AbstractInvocation.cs

```diff
@@ -137,6 +139,7 @@ public IInvocationProceedInfo CaptureProceedInfo()
 
 		protected abstract void InvokeMethodOnTarget();
 
+		[DoesNotReturn]
```

Comment:
(For some strange reason, PublicApiGenerator decided not to fully qualify this attribute type name with its namespace. No big deal, but we may get an unexpected `ref/` file change due to this if ever we upgrade PublicApiGenerator.)